docs/podman-remote.1.md

```diff
@@ -0,0 +1,139 @@
+| [podman-kill(1)](podman-kill.1.md)               | Kill the main process in one or more containers.                            |
+| [podman-load(1)](podman-load.1.md)               | Load an image from a container image archive into container storage.        |
+| [podman-logs(1)](podman-logs.1.md)               | Display the logs of a container.                                            |
+| [podman-pause(1)](podman-pause.1.md)             | Pause one or more containers.                                               |
+| [podman-pod(1)](podman-pod.1.md)                 | Management tool for groups of containers, called pods.                      |
+| [podman-port(1)](podman-port.1.md)               | List port mappings for a container.                                         |
+| [podman-ps(1)](podman-ps.1.md)                   | Prints out information about containers.                                    |
+| [podman-pull(1)](podman-pull.1.md)               | Pull an image from a registry.                                              |
+| [podman-push(1)](podman-push.1.md)               | Push an image from local storage to elsewhere.                              |
+| [podman-restart(1)](podman-restart.1.md)         | Restart one or more containers.                                             |
+| [podman-rm(1)](podman-rm.1.md)                   | Remove one or more containers.                                              |
+| [podman-rmi(1)](podman-rmi.1.md)                 | Removes one or more locally stored images.                                  |
+| [podman-run(1)](podman-run.1.md)                 | Run a command in a new container.                                           |
+| [podman-save(1)](podman-save.1.md)               | Save an image to a container archive.                                       |
+| [podman-start(1)](podman-start.1.md)             | Start one or more containers.                                               |
+| [podman-stop(1)](podman-stop.1.md)               | Stop one or more running containers.                                        |
+| [podman-system(1)](podman-system.1.md)           | Manage podman.                                                              |
+| [podman-tag(1)](podman-tag.1.md)                 | Add an additional name to a local image.                                    |
+| [podman-top(1)](podman-top.1.md)                 | Display the running processes of a container.                               |
+| [podman-unpause(1)](podman-unpause.1.md)         | Unpause one or more containers.                                             |
+| [podman-version(1)](podman-version.1.md)         | Display the Podman version information.                                     |
+| [podman-volume(1)](podman-volume.1.md)           | Manage Volumes.                                                             |
+
```

Comment:
Really wish there was an easy way to include files in md.  I just worry that we'll forget to update the section in this page or the one in podman.1.md at some point.